### PR TITLE
Add support for random_seed

### PR DESCRIPTION
**Pull request by kornholi**

### CODE REVIEW

**Comment by hrfuller** [comment]
Does the default random seed (current timestamp) not work for your case?

**Comment by kornholi** [comment]
I don't think there's a default right now? Either way it's nice to have reproducibility for some of the things I'm working on.

**Comment by hrfuller** [comment]
Elasticsearch uses the current timestamp as default if no seed is given. I have some hesitation about about this being a good external facing feature. This is for performance testing right? Could you use the REST endpoints instead?

File: descarteslabs/services/metadata.py [hunk 1]
```diff
@@ -493,7 +501,7 @@ def keys(self, products=None, const_id=None, sat_id=None, date='acquired', place
                              end_time=end_time, cloud_fraction=cloud_fraction,
                              cloud_fraction_0=cloud_fraction_0, fill_fraction=fill_fraction,
                              params=params, limit=limit, offset=offset, fields=["key"], dltile=dltile,
-                             sort_field=sort_field, sort_order=sort_order)
+                             sort_field=sort_field, sort_order=sort_order, randomize=None)
```
Comment (hrfuller) [inline descarteslabs/services/metadata.py]: This should be `randomize=randomize` right? do you not want to allow randomized keys?

**Pull request merged**